README.md

```diff
@@ -1,2 +1,62 @@
 # daedalus-deploy
 Deploy tool for the Daedalus app
+
+## Requirements
+
+1. [Python3](https://www.python.org/downloads/) (>= 3.10)
+2. [Hatch](https://hatch.pypa.io/latest/install/)
+
+## Installation
+
+This project is built with hatch. It is not published to PyPI, so must be built and installed from local sources.
+Clone this repo then:
+```
+hatch build 
+pip3 install dist/daedalus_deploy-{version}.tar.gz
+```
+replacing `{version}` with the current version (`hatch version`).
+
+If you have a newer version of pip3, you may need to create a suitable virtual environment for installing the tool:
+```
+python3 -m venv daedalus-venv
+```
+...and activate it:
+```
+source daedalus-venv/bin/activate
+```
+(You can deactivate the venv later if required with `deactivate`.)
+
+## Usage
+
+```
+Usage:
+  daedalus start [--pull] [<configname>]
+  daedalus stop  [--volumes] [--network] [--kill] [--force]
+  daedalus destroy
+  daedalus status
+  daedalus upgrade
+
+Options:
+  --pull                    Pull images before starting
+  --volumes                 Remove volumes (WARNING: irreversible data loss)
+  --network                 Remove network
+  --kill                    Kill the containers (faster, but possible db corruption)
+```
+
+Once a configuration is set during `start`, it will be reused by subsequent commands
+(`stop`, `status`, `upgrade`, `user`, etc) and removed during destroy.
+The configuration usage information is stored in `config/.last_deploy.`
+
+## Deployment configurations
+There are three configurations
+- fakeproxy: for local testing
+- staging: for deployment onto our staging server at `daedalus-dev.dide.ic.ac.uk`
+- prod: on the way!
```

Comment:
When I run

```
hatch shell
daedalus start fakeproxy
```

I get an error about not finding config/config.yml

```
Traceback (most recent call last):
  File "/home/rfitzjoh/.local/share/hatch/env/virtual/daedalus-deploy/pdhgqYt3/daedalus-deploy/bin/daedalus", line 8, in <module>
    sys.exit(main())
  File "/home/rfitzjoh/Documents/Projects/eppi/daedalus-deploy/src/daedalus_deploy/cli.py", line 95, in main
    config_name, cfg = load_config(path, config_name, options)
  File "/home/rfitzjoh/Documents/Projects/eppi/daedalus-deploy/src/daedalus_deploy/cli.py", line 75, in load_config
    cfg = DaedalusConfig(path, dat["config_name"], options=options)
  File "/home/rfitzjoh/Documents/Projects/eppi/daedalus-deploy/src/daedalus_deploy/config.py", line 8, in __init__
    dat = config.config_build(path, dat, config_name, options=options)
  File "/home/rfitzjoh/.local/share/hatch/env/virtual/daedalus-deploy/pdhgqYt3/daedalus-deploy/lib/python3.10/site-packages/constellation/config.py", line 21, in config_build
    data_extra = read_yaml("{}/{}.yml".format(path, extra))
  File "/home/rfitzjoh/.local/share/hatch/env/virtual/daedalus-deploy/pdhgqYt3/daedalus-deploy/lib/python3.10/site-packages/constellation/config.py", line 12, in read_yaml
    with open(filename, "r") as f:
FileNotFoundError: [Errno 2] No such file or directory: 'config/config.yml'
```

Reply:
I'm still quite confused about this. I just committed the toml change, deleted the local dir to get rid of any old dists and venvs, cloned a fresh copy, and did `hatch shell` then `daedalus start fakeproxy` and it worked fine..
And it should never be looking for "config.yml" unless it's somehow got its args confused. 
Maybe we can talk about this later too!